kube_operator/client.go

```diff
@@ -333,7 +337,22 @@ func getK8sObjectFromYaml(yamlFiles []YamlFile, sch *runtime.Scheme) ([]APIObjec
 	_ = v1scheme.AddToScheme(sch)
 	_ = scheme.AddToScheme(sch)
 
-	for _, fileStr := range yamlFiles {
+	// multiple yaml files can be in one file separated by '---'
+	// these are split here and rejoined with the single files
```

Comment:
i really dislike yaml...the person who thought this was a good idea should have their coding license revoked.

Reply:
can't trust anything that puts so much syntactic significance on whitespace